content/pages/constitution.md

```diff
@@ -211,19 +211,44 @@ Committee office, M3 1003.
     2. The current committee will then select evaluate the applications
        received and select applicants for interviews;
 
-    3. The current committee will select interviewers. Interviewers should be
-       chosen by the current committee, and are expected to disclose any
+    3. A single interview shall be held for each candidate. Interviews shall
+       not exceed 30 minutes in length;
+
+    4. The current committee will select interviewers. Two interviewers should
+       be chosen by the current committee, and are expected to disclose any
        conflicts of interest when presented with the list of candidates. If a
        conflict of interest arises, an alternate interviewer shall be chosen.
        In any case, if no interviewer is explicitly chosen and there is no
-       conflict of interest, the current Chair shall conduct the interviews;
+       conflict of interest, the current Chair and Secretary shall conduct the
+       interviews;
+
+    5. Two interviewers must be present at every interview, and each shall
```

Comment:
There is one last benefit I should have mentioned explicitly about there being two interviewers, which is that the presence of the second person holds the interviewer more accountable. It's not just about collecting two sets of feedback; in my experience, having another interviewer observe or participate in the interview also improves the quality of the feedback produced by both interviewers. I think this is more fair to the candidate, because otherwise they are the only person holding an interviewer accountable and the only individual able to provide feedback back to us on our interview performance.